yb-voyager/cmd/import.go

```diff
@@ -128,7 +129,7 @@ func registerTargetDBConnFlags(cmd *cobra.Command) {
 		"provide target SSL Certificate Path")
 
 	cmd.Flags().StringVar(&tconf.SSLMode, "target-ssl-mode", "prefer",
-		"specify the target SSL mode out of - disable, allow, prefer, require, verify-ca, verify-full")
+		"specify the target SSL mode: (disable, allow, prefer, require, verify-ca, verify-full)")
```

Comment:
Should we also include a link to any page (YB/PG) that describes these options in detail?